gobblin-core/src/test/java/gobblin/converter/csv/PercentageDoubleStringConverterTest.java

```diff
@@ -0,0 +1,36 @@
+package gobblin.converter.csv;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import gobblin.converter.DataConversionException;
+
+
+@Test(groups = {"gobblin.converter"})
+public class PercentageDoubleStringConverterTest {
+  public void convertOutput()
+      throws IOException, DataConversionException {
+    JsonParser parser = new JsonParser();
+    JsonElement jsonElement = parser
```

Comment:
Really, it got checked in here. https://github.com/enjoyear/gobblin/blob/google-adwords/gobblin-core/src/test/resources/converter/csv/schema_with_10_fields.json